api/api.pb.txt

```diff
@@ -4065,6 +4077,14 @@ file {
         oneof_index: 0
         json_name: "registry"
       }
+      field {
+        name: "Config"
+        number: 3
+        label: LABEL_OPTIONAL
+        type: TYPE_STRING
+        oneof_index: 0
+        json_name: "Config"
```

Comment:
Same for the `json_name`

This feature hasn't shipped yet in a Docker release, so if we want to (and can) change it, now is the time 🤗

Reply:
(perhaps it's because this one has to be exported (e.g.), but double checking)

Reply:
Fixed in https://github.com/docker/swarmkit/pull/2813